temporalcli/commandsmd/commands.md

```diff
@@ -570,7 +570,26 @@ TODO
 
 ### temporal workflow delete: Deletes a Workflow Execution.
 
-TODO
+The `temporal workflow delete` command is used to delete a specific [Workflow Execution](/concepts/what-is-a-workflow-execution).
+This asynchronously deletes a workflow's [Event History](/concepts/what-is-an-event-history).
+If the [Workflow Execution](/concepts/what-is-a-workflow-execution) is Running, it will be terminated before deletion.
+
+```
+temporal workflow delete \
+		--workflow-id MyWorkflowId \
+```
+
+Use the options listed below to change the command's behavior.
+
+#### Options set for single workflow or batch:
+
+* `--workflow-id`, `-w` (string) - Workflow Id. Either this or query must be set.
+* `--run-id`, `-r` (string) - Run Id. Cannot be set when query is set.
+* `--query`, `-q` (string) - Start a batch to operate on Workflow Executions with given List Filter. Either this or
+  Workflow Id must be set.
+* `--reason` (string) - Reason to perform batch. Only allowed if query is present unless the command specifies
+  otherwise. Defaults to message with the current user's name.
+* `--yes`, `-y` (bool) - Confirm prompt to perform batch. Only allowed if query is present.
```

Comment:
```suggestion
#### Options

Includes options set for [single workflow or batch](#options-set-single-workflow-or-batch)
```

Should be able to just reference the existing one (we should probably add parser checks to ensure there are not two options sets for the same name)